README.md

```diff
@@ -38,14 +38,14 @@ wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh |
 ```
 Close current terminal and open a new one or `source ~/.bashrc`
 ```
-nvm install 10.13.0
+nvm install 16.14.0
 ```
 Optionally, install a global Angular CLI in order to execute `ng` commands without prepending `npx`.   
-Otherwise, prepend `npx` to every command in this README if a global @angular/cli was not installed.
+Otherwise, prepend `npx` to every `ng` command in this README if a global @angular/cli was not installed.
 Before installing, follow https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-two-change-npms-default-directory to fix permissions if needed.
 
 ```
-$npm i -g @angular/cli@1.3.1		
+$npm i -g @angular/cli@13.2.2    
```

Comment:
We should just remove this entirely, and rely on the Angular installed via package.json.